src/styles/auto-prefix.js

```diff
@@ -3,6 +3,15 @@ import warning from 'warning';
 
 const prefixers = {};
 
+const pruneNulls = style => {
+  Object.keys(style).forEach(key => {
+    if (style[key] === null || style[key] === undefined) {
+      delete style[key];
```

Comment:
Yeah! I was looking for the link that you posted where I read this but couldn't find it. I think it was from Google but I don't remember why.

The irony is this:
1. If I'm not mistaken, I think their recommendation was to just set it as null or undefined instead, which is what this PR is fixing
2. In the inline-style-prefixer, they're also using `delete` to delete properties.